zcash_primitives/src/transaction/fees.rs

```diff
@@ -60,7 +62,7 @@ pub trait FutureFeeRule: FeeRule {
 }
 
 /// An enumeration of the standard fee rules supported by the wallet.
-#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
This should also only need to be serialized via the `Proposal` protobuf encoding.